README.md

```diff
@@ -29,69 +54,191 @@ let extensionHandler = {
+        reqObj,
+        req.query.state,
+        req.query.code,
+        fdkClient.extension,
+        sessionId
+    );
+    // Return redirect url obtained in fpInstall call
+    // Return fdk session (for example in cookies or jwt token or any other form )
+    /*res.cookie(compCookieName, fdkSession.id, {
+        secure: true,
+        httpOnly: true,
+        expires: fdkSession.expires,
+        signed: true,
+        sameSite: "None",
+    });
+    res.redirect(redirectUrl); */
+});
+
+router.post("/fp/auto_install", async (req, res, next) => {
+    await handlers.fpAutoInstall(
+        reqObj,
+        req.body.company_id,
+        req.body.code,
+        fdkClient.extension
+    );
+    res.json({ message: "success" });
+});
+
+router.post("/fp/uninstall", async (req, res, next) => {
+    await handlers.fpUninstall(
+        reqObj,
+        req.body.company_id,
+        fdkClient.extension
+    );
+    res.json({ success: true });
 });
-app.use(fdkClient.fdkHandler);
 
-app.listen(8080);
+app.use(router);
+app.listen(3000);
 ```
+### Parameters of setupFDK function
+#### Parameter table for `setupFdk` function
+
+| Parameter  | Description | 
+| ------------- | ------------- |
+| api_key  | API Key of an extension for authentication.  |
+| api_secret  | API Secret of an extension for authentication.  |
+| base_url?  | Base URL for extension.  |
+| scopes?  | An array of scopes indicating the specific permissions needed for an extension. |
+| callbacks  | The callback function to handle extension-related tasks. |
+| storage  | An instance of storage (e.g., RedisStorage) for data storage.  |
+| access_mode  | Access mode of an extension. It can be `online` or `offline`.  |
+| cluster?  | The API url of the Fynd Platform cluster. |
+| debug?  | Enable debug logs if it is `true`. Value can be `true` or `false`.  |
+| [webhook_config](https://github.com/gofynd/fdk-extension-javascript?tab=readme-ov-file#parameter-table-for-webhook-configuration)  | Necessary configuration for webhooks.  |
+
+#### Parameter table for `webhook` configuration
+| Parameter  | Description |
+| ------------- | ------------- |
+| api_path  | API endpoint to process webhooks event.  |
+| notification_email  | Email address for webhook related notifications.  |
+| subscribe_on_install?  | Whether to auto subscribe to all webhooks on extension installation. It can be true or false. |
+| subscribed_saleschannel?  | If `specific` then you have to manually subscribe to sales channel/website level events for individual sales channels. Value can be `all` or `optional`. |
+| [event_map](https://github.com/gofynd/fdk-extension-javascript?tab=readme-ov-file#parameter-table-for-event_map-object)  | A mapping of events to corresponding handlers for webhook processing.  |
+
+#### Parameter table for `event_map` object
+| Parameter  | Description |
+| ------------- | ------------- |
+| key  | API endpoint to process webhooks event.  |
```

Comment:
@jinalviranii  I don't think key is here referring to Webhook endpoint, please cross check once